commands/utils/params.go

```diff
@@ -209,7 +217,7 @@ func extractGitParamsFromEnv() (Git, error) {
 	if err = readParamFromEnv(GitTokenEnv, &gitParams.Token); err != nil {
 		return Git{}, err
 	}
-
+	_ = readParamFromEnv(GitUsernameEnv, &gitParams.Username)
```

Comment:
Let's add a comment, explaining why we're ignoring the error returned,
The same goes for the other calls to this function where we ignore the error.

Reply:
done